browsers/brave/brave.md

```diff
@@ -0,0 +1,27 @@
+# Running WebContainer in Brave Browser
+
+WebContainer — the technology behind Node.js projects on StackBlitz — does work in Brave. However, it requires a small configuration change.
+
+## What’s the problem?
+
+By default, Brave’s “Shields” feature blocks [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) and cookies from third-party domains.
+
+When you visit a WebContainer-enabled project on [stackblitz.com](https://www.stackblitz.com/), StackBlitz tries to register a Service Worker for the domain `<project_name>.stackblitz.io`. Since it’s considered a third-party domain, Brave will reject it.
+
+## How to allow third-party service workers for stackblitz.com
+
+1. Visit a WebContainer-based project, for instance https://www.stackblitz.com/edit/nextjs. The project’s boot sequence might stay stuck on the “Running start command” step:
+
+<img src="1-stuck-project.png" width="784" alt="Screenshot of Brave on a WebContainer project with the Brave Shields feature on. Loading the project’s web server is stuck on the last step.">
+
+2. Click on the “Shields” icon at the right of the address bar, then click on “Advanced View”.
+
+<img src="2-shields-popup.png" width="380" alt="">
```

Comment:
Maybe fill in the alt? Same for screenshot 3.

Reply:
I thought about it, but the alt text would be redundant with the text before, so it's better to leave it empty in that case.

Empty alt text indicates that an image is decorative or redundant with the textual information in its context.

Reply:
You know what, we could remove all alt text in this page or have alt text for all images, both work IMO. It's a style question, minimal vs more verbose.

Let’s try alt text for all images.